modules/serviceregistry/src/test/java/org/opencastproject/serviceregistry/impl/ServiceRegistryJpaImplTest.java

```diff
@@ -401,19 +408,20 @@ public void testDispatchingJobsHigherMaxLoad() throws Exception {
     properties.put("dispatchinterval", "1000");
     serviceRegistryJpaImpl.updated(properties);
     registerTestHostAndService();
-    Job testJob = serviceRegistryJpaImpl.createJob(TEST_HOST, TEST_SERVICE, TEST_OPERATION, null, null, true, null,
+    Job testJob = serviceRegistryJpaImpl.createJob(TEST_HOST, TEST_SERVICE_FAIRNESS, TEST_OPERATION, null, null, true,
+            null,
             10.0f);
```

Comment:
@gregorydlogan, I changed this to use TEST_SERVICE_FAIRNESS because the mock http client always return 503 for TEST_SERVICE - TEST_PATH.